config/config.go

```diff
@@ -269,6 +269,7 @@ type Config struct {
 	ClusterRatelimitMaxGroupShards int `yaml:"cluster-ratelimit-max-group-shards"`
 
 	LuaModules *listFlag `yaml:"lua-modules"`
+	LuaSources string    `yaml:"lua-sources"`
```

Comment:
Should be listFlag I think

Reply:
Lets use listFlag as it supports proper splitting and handling of empty value

Reply:
done